model/patch_lifecycle.go

```diff
@@ -554,27 +575,28 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 	}
 
 	patchVersion := &Version{
-		Id:                   p.Id.Hex(),
-		CreateTime:           time.Now(),
-		Identifier:           p.Project,
-		Revision:             p.Githash,
-		Author:               p.Author,
-		Message:              p.Description,
-		BuildIds:             []string{},
-		BuildVariants:        []VersionBuildStatus{},
-		Status:               evergreen.PatchCreated,
-		Requester:            requester,
-		ParentPatchID:        p.Triggers.ParentPatch,
-		ParentPatchNumber:    parentPatchNumber,
-		Branch:               projectRef.Branch,
-		RevisionOrderNumber:  p.PatchNumber,
-		AuthorID:             p.Author,
-		Parameters:           params,
-		Activated:            utility.TruePtr(),
-		AuthorEmail:          authorEmail,
-		ProjectStorageMethod: evergreen.ProjectStorageMethodDB,
-	}
-	intermediateProject.CreateTime = patchVersion.CreateTime
+		Id:                  p.Id.Hex(),
+		CreateTime:          time.Now(),
+		Identifier:          p.Project,
+		Revision:            p.Githash,
+		Author:              p.Author,
+		Message:             p.Description,
+		BuildIds:            []string{},
+		BuildVariants:       []VersionBuildStatus{},
+		Status:              evergreen.PatchCreated,
+		Requester:           requester,
+		ParentPatchID:       p.Triggers.ParentPatch,
+		ParentPatchNumber:   parentPatchNumber,
+		Branch:              projectRef.Branch,
+		RevisionOrderNumber: p.PatchNumber,
+		AuthorID:            p.Author,
+		Parameters:          params,
+		Activated:           utility.TruePtr(),
+		AuthorEmail:         authorEmail,
+	}
+	if mustInsertParserProjectDuringFinalization {
+		intermediateProject.CreateTime = patchVersion.CreateTime
```

Comment:
Same comment here, I think this could be moved to L703

Reply:
Done.